src/styles/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  margin-left: 16px;
+  box-sizing: border-box;
+
+  &__star {
+    background-image: url(/images/star.svg);
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__star:not(:first-child) {
+    margin-left: 4px;
+  }
+
+  &--0 &__star:nth-child(-n + 4) {
+    background-image: url(/images/star-active.svg);
```

Comment:
Stars should display the correct amount of active stars according to the modifier
![image](https://github.com/mate-academy/layout_product-cards/assets/96409563/9a567f93-8b3d-4255-a7e3-4d162cbfe805)